content/guides/cloud/smart-orchestration.md

```diff
@@ -99,48 +97,47 @@ failure will:
 
```

Comment:
Good question!

Reply:
I think the Spec Prioritization section has the "Consider the Scenario" alert instead and we don't want to drown the user in alerts. But I agree it would be better to align the two sections. Will circle back.